charmander/tests/routes/pizza/test_crud.py

```diff
@@ -0,0 +1,153 @@
+"""
+Pizza CRUD routes tests.
+
+Tests are sunny day cases under the assumption that framework conventions
+handle most error conditions.
+
+"""
+from unittest.mock import patch
+
+from hamcrest import (
+    assert_that,
+    contains,
+    equal_to,
+    has_entries,
+    is_,
+)
+from microcosm_postgres.context import SessionContext, transaction
+from microcosm_postgres.identifiers import new_object_id
+from microcosm_postgres.operations import recreate_all
+
+from charmander.app import create_app
+from charmander.models.pizza_model import (
+    CrustType,
+    Pizza,
+    PizzaSize,
+)
+
+
+class TestPizzaRoutes:
+
+    def setup(self):
+        self.graph = create_app(testing=True)
+        self.client = self.graph.flask.test_client()
+        self.uri = "/api/v1/pizza"
+        recreate_all(self.graph)
+
+        self.pizza = Pizza(
+            id=new_object_id(),
+            customer_id=new_object_id(),
+            crust_type=CrustType.CHEESE_STUFFED,
+            size=PizzaSize.LARGE,
+        )
+
+    def teardown(self):
+        self.graph.postgres.dispose()
```

Comment:
These tests use a real db to be able to test any nontrivial/store-specific logic as early as possible.